__tests__/TestCallOptions.test.ts

```diff
@@ -25,25 +25,28 @@ describe('ContractCallOptions', () => {
   });
 
   describe('#Promises', () => {
-    it('mints long tokens with ETH without promise', async () => {
+    it.only('mints long tokens with ETH without promise', async () => {
```

Comment:
❌ 

Also we should make it so this is caught by the linter. See https://github.com/dydxprotocol/node-service-lib-dev/blob/master/.eslintrc.yaml#L17

Reply:
added `tslint-no-focused-test` to the linter circle ci